skimage/measure/tests/test_simple_metrics.py

```diff
@@ -75,3 +74,23 @@ def test_NRMSE_errors():
     # invalid normalization name
     with testing.raises(ValueError):
         compare_nrmse(x, x, 'foo')
+
+
+def test_EME_greyscale():
```

Comment:
Does the original paper provide any absolute values for given images that we can try and reproduce?  Only checking whether the value goes down after enhancement is not quite strict enough.

Reply:
This came up earlier (see [here](https://github.com/scikit-image/scikit-image/pull/3776#discussion_r263983133)). The image in question could be a good candidate for comparison. Unfortunately its source isn't mentioned inside the paper and the one found through searching may not be *exactly* the same.

Another issue is that the absolute value may change depending on `eps`. That parameter can help to avoid division by zero but is an addition of this PR and not part of the paper.

Reply:
Furthermore the rescaling between 0 and 1 isn't part of the paper either, so I doubt we'd get comparable absolute values even if we used the same image as in the paper.

Reply:
That's fine; I was hoping for some rough agreement between their work and ours, otherwise how do we know we're even in the right ballpark?  Overall, any test that we can define that is a bit more strict will be helpful.

Reply:
@stefanv @lagru I looked at the experiment section of the paper, I managed to find the moon surface image [here](https://homepages.inf.ed.ac.uk/rbf/HIPR2/images/moo1.gif), running the code with that image without converting to float and (0,1)-rescaling, I can get the value to be ~6.67 if the `size` is 6 and `eps` is `1e-8`. The same parameters yield ~6.28 when converted to float and rescaled.

Reply:
Does this correspond to the values in the paper?

Reply:
Yes, the paper lists 6.78 on page 4, Figure 1b.

Reply:
OK, so we can tighten up one of the tests to match against a specific value.

Why are the values different depending on the dtype and scaling?

Reply:
I need to think some more about it, right now I am not sure why this is the case.